x-pack/plugins/security/server/security_checkup/check_cluster_data.ts

```diff
@@ -13,18 +13,22 @@ export const createClusterDataCheck = () => {
   return async function doesClusterHaveUserData(esClient: ElasticsearchClient, log: Logger) {
     if (!clusterHasUserData) {
       try {
-        const indices = await esClient.cat.indices({
-          format: 'json',
-          h: ['index', 'docs.count'],
-        });
-        clusterHasUserData = indices.some((indexCount) => {
+        const { indices = {} } = await esClient.indices.stats();
+        const indexIds = indices ? Object.keys(indices) : [];
+
+        clusterHasUserData = indexIds.some((indexName: string) => {
```

Comment:
__optional nit:__ it feels a bit weird that the item of the `indexIds` array named as `indexName` and not `indexId`? :slightly_smiling_face: